sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/enums.py

```diff
@@ -50,3 +50,11 @@ class KeyCurveName(str, Enum):
     p_384 = "P-384"  #: The NIST P-384 elliptic curve, AKA SECG curve SECP384R1.
     p_521 = "P-521"  #: The NIST P-521 elliptic curve, AKA SECG curve SECP521R1.
     p_256_k = "P-256K"  #: The SECG SECP256K1 elliptic curve.
+
+
+class WellKnownIssuerNames(str, Enum):
+    """Collection of well-known issuer names"""
+
+    Self = "Self"  #: Use this issuer for a self-signed certificate
+    Unknown = "Unknown"  #: If you use this, you must manually get the cert yourself from the issuer of your choice.
+                         #: Must call :func:`merge_certificate` to complete creation.
```

Comment:
Does sphinx render the two-line comment properly?

Reply:
no it doesn't, i forgot to check. I've tried to condense it into one line